jetty-webapp/src/main/java/org/eclipse/jetty/webapp/CachingWebAppClassLoader.java

```diff
@@ -86,6 +102,27 @@ public URL getResource(String name)
         return url;
     }
 
+    @Override
+    protected URL getResourceClassReference(String name)
```

Comment:
The `URL findResource(String name)` method should be overridden for this, not a new method.